docs/readme.md

```diff
@@ -17,3 +17,35 @@ items:
 # Welcome to DEV's developer documentation
 
 Here you can find instructions on how to setup your own local copy of DEV, how to navigate the code, how to contribute, and how to troubleshoot issues, among other things.
+
+# Running the documentation locally
+
+DEV's documentation is built with [GitDocs NodeJS library](https://www.npmjs.com/package/gitdocs).
+
+The first step to running the documentations it locally is to install the `GitDocs` package globally.
+
```

Comment:
Why not just link to the contributing file? This way we avoid having to update the doc twice when we have to change it.

Reply:
I would agree, but I think that by convention the README should include steps to run a specific section of code.

I'd say we should remove it from contributing, but people who visit docs.dev.to will need those instructions too.

Reply:
The reason why I bring this up it is because we had two sections of "contributing" already in the past and they weren't aligned: https://github.com/thepracticaldev/dev.to/pull/3659

in my experience is easier to maintain only one version of the docs :D

Reply:
I agree with that, but I feel the README really should explain at least how to run the docs. This is an issue that @nickytonline has already mentioned encountering. I think this is necessary, even though it's not ideal.

Reply:
Agreed